transaction-service/src/main/kotlin/protobuf/GRPCService.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package protobuf;
+
+import "ProtoMessage.proto";
+
+service TransactionsService{
+  rpc getTransactions (TransactionRequestProto) returns (stream TransactionResponseProto);
```

Comment:
if your service returns many elements at once and then does nothing - it is not a server-side stream. In such case, it should be a plain request-reply where the reply contains a list of elements
The stream should be used when the server will publish updates to stream in the future

Reply:
changed to list